jwala-services/src/test/java/com/cerner/jwala/service/group/impl/GroupServiceImplVerifyTest.java

```diff
@@ -59,17 +61,28 @@ public GroupServiceImplVerifyTest() {
     @Before
     public void setUp() {
         user = new User("unused");
+        reset(Config.mockGroupPersistenceService);
     }
 
     @Test
     public void testCreateGroup() {
 
-        final CreateGroupRequest command = mock(CreateGroupRequest.class);
+        final CreateGroupRequest command = new CreateGroupRequest("Group");
+        when(Config.mockGroupPersistenceService.getGroup(anyString())).thenThrow(NotFoundException.class);
+        groupService.createGroup(command,
+                user);
+
+        verify(Config.mockGroupPersistenceService, times(1)).createGroup(command);
```

Comment:
This is minor, just an FYI - verify's default Times is 1 so you can omit it in the future

    verify(someMock).doSomething(...)